boards/arduino-nano-33-ble-sense/board.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C)  2024 Méwen Berthelot
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_arduino-nano-33-ble-sense
+ * @{
+ *
+ * @file
+ * @brief       Board specific initialization for Arduino Nano 33 BLE sense
+ *
+ * @author      Méwen Berthelot <berthelotmewen@gmail.com>
+ *
+ * @}
+ */
+
+#include "board.h"
+
+#if defined(APDS9960) || defined(HTS221) || defined(lps22hb)
```

Comment:
```suggestion
#if defined(MODULE_APDS9960) || defined(MODULE_HTS221) || defined(MODULE_LPS22HB)
```

that's what I would expect, but not fully sure. Same below.

Reply:
That's right I don't know why I wrote this that way its absolutely not correct

Reply:
Which makes me wonder whether it is actually needed, since it should have evaluated to false as is.

Reply:
Can you confirm it is really needed, or did it work with the old ifdef too?